src/driver.jl

```diff
@@ -0,0 +1,53 @@
+struct Graph500Results{T<:Integer}
+    g::Graph{T}
+    SCALE::Integer
+    NBFS::Integer
+    kernel_1_time::Float64
+    kernel_2_time::SharedArray{Float64,1}
+    kernel_2_nedge::SharedArray{Int64,1}
+end
+
+function driver(
+  SCALE::Integer,
+  edgefactor::Integer;
+  replicate::Bool=false,
+  seed::AbstractArray=[])
+
+    # generate edge list
+    ij = kronecker_generator(SCALE, edgefactor;replicate=replicate,seed=seed)
+
+    # timing kernel 1
+    tic()
+    g = kernel_1(ij)
+    kernel_1_time = toq()
+
+    # For computing the graph element type
+    T = eltype(g)
+
+    search_key = key_sampling(g)
+    NBFS = length(search_key)
+
+    kernel_2_time =  zeros(Float64,NBFS)
+    kernel_2_nedge = zeros(Float64,NBFS)
+
+    for k in 1:NBFS
+        tic()
```

Comment:
This won't be needed when I replace current kernel_2 with a call to `bfs_parent`